[Pytorch][Bootcamp] Add fix and testing for non-vectorized Adadelta optimizer to handle complex numbers

choward232 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #66588
* __->__ #66587

Made some changes in the step function of the non-vectorized Adadelta optimizer to handle complex numbers as two real numbers as per 65711 on github

Differential Revision: [D31630069](https://our.internmc.facebook.com/intern/diff/D31630069/)

Commit:
[Pytorch][Bootcamp] Add fix and testing for non-vectorized Adadelta optimizer to handle complex numbers

Made some changes in the step function of the non-vectorized Adadelta optimizer to handle complex numbers as two real numbers as per 65711 on github

Differential Revision: [D31630069](https://our.internmc.facebook.com/intern/diff/D31630069/)

[ghstack-poisoned]

Commit:
Update on "[Pytorch][Bootcamp] Add fix and testing for non-vectorized Adadelta optimizer to handle complex numbers"


Made some changes in the step function of the non-vectorized Adadelta optimizer to handle complex numbers as two real numbers as per 65711 on github

Differential Revision: [D31630069](https://our.internmc.facebook.com/intern/diff/D31630069/)

[ghstack-poisoned]

albanD reviewed (commented):
nit: un-used variables otherwise ok.

Commit:
Update on "[Pytorch][Bootcamp] Add fix and testing for non-vectorized Adadelta optimizer to handle complex numbers"


Made some changes in the step function of the non-vectorized Adadelta optimizer to handle complex numbers as two real numbers as per 65711 on github

Differential Revision: [D31630069](https://our.internmc.facebook.com/intern/diff/D31630069/)

[ghstack-poisoned]

albanD reviewed (approved):
Nice!

Commit:
Update on "[Pytorch][Bootcamp] Add fix and testing for non-vectorized Adadelta optimizer to handle complex numbers"


Made some changes in the step function of the non-vectorized Adadelta optimizer to handle complex numbers as two real numbers as per 65711 on github

Differential Revision: [D31630069](https://our.internmc.facebook.com/intern/diff/D31630069/)

[ghstack-poisoned]
